apps/condo/domains/resident/access/Resident.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Generated by `createschema resident.Resident 'user:Relationship:User:CASCADE; organization:Relationship:Organization:PROTECT; property:Relationship:Property:PROTECT; billingAccount?:Relationship:BillingAccount:SET_NULL; unitName:Text;'`
+ */
+
+async function canReadResidents ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    if (user.type === 'resident') {
+        return {
+            user: { id: user.id },
+        }
+    }
+    return {
+        organization: { employees_some: { user: { id: user.id }, isBlocked: false } },
```

Comment:
Organization employees don't have access to residents

Reply:
deteledAt: null

Reply:
Looks, like `deletedAt: null` condition part is missing in 9 places.
Created separate task for it.

Reply:
Revoked permission in `b99b77c8`, made some fixes of tests in `b4c282f0`, added task SBERDOMA-877 for global tests refactoring.